scene/3d/vehicle_body.cpp

```diff
@@ -559,13 +559,13 @@ void VehicleBody::_resolve_single_bilateral(PhysicsDirectBodyState *s, const Vec
 			b2invinertia,
 			b2invmass);
 
-	real_t rel_vel = jac.getRelativeVelocity(
+	jac.getRelativeVelocity(
 			s->get_linear_velocity(),
 			s->get_transform().basis.transposed().xform(s->get_angular_velocity()),
 			b2lv,
 			b2trans.xform(b2av));
 
-	rel_vel = normal.dot(vel);
+	real_t rel_vel = normal.dot(vel);
```

Comment:
I think you're hiding a bug here. `getRelativeVelocity()` is just a getter so not assigning its result to a variable is pointless.

I guess what was intended is `rel_vel = normal.dot(rel_vel);`.